### PR TITLE
feat(table-wrapper): add default value for table models

### PR DESCRIPTION
**Pull request by Yax94**

### CODE REVIEW

File: src/pythie_serving/table_wrapper.py [hunk 1]
```diff
@@ -44,25 +44,40 @@ def _create_model_specs(self, model_config: ModelConfig) -> ModelSpecs:
...
-        }
+        extra_specs = None
+        if "default_value" in metadata:
+            default_value = metadata["default_value"]
+            if type(default_value) != table_type_mapping[metadata["target_name"]]:
+                raise PythieServingException(
+                    f"Can't assign default_value {default_value} because it's type {type(default_value)} is "
+                    f"different from target type {table_type_mapping[metadata['target_name']]}"
+                )
+            extra_specs = {"default_value": default_value}
```
Comment (matbleu) [inline src/pythie_serving/table_wrapper.py:55]: You could move that to a new method `_parse_extra_specs` to make it easier to read

File: src/pythie_serving/table_wrapper.py [hunk 2]
```diff
@@ -44,25 +44,40 @@ def _create_model_specs(self, model_config: ModelConfig) -> ModelSpecs:
...
+        )
 
     def _predict(self, model_specs: ModelSpecs, samples: NDArray) -> NDArray:
 
         output = np.empty((np.shape(samples)[0],), dtype=int)
+        found_values = 0
         for idx, sample in enumerate(samples):
             try:
                 pred = model_specs["model"][tuple(feature_value for feature_value in sample)]
+                found_values += 1
```
Comment (matbleu) [inline src/pythie_serving/table_wrapper.py:72]: Looks like `found_values` is not used. Could you remove it ?

**Review by matbleu — CHANGES_REQUESTED** [review]

File: src/pythie_serving/table_wrapper.py [hunk 3]
```diff
@@ -19,6 +20,18 @@
 class TablePredictionServiceServicer(AbstractPythieServingPredictionServiceServicer):
     model_file_extension = ".csv"
 
+    def _parse_extra_specs(self, metadata: dict[str, Any], table_type_mapping: dict[str, Any]):
```
Comment (matbleu) [inline src/pythie_serving/table_wrapper.py:23]: Missing a return type, but should have been catched by the CI ...
Could you still add it ?

**Review by matbleu — COMMENTED** [review]
all good for me, except this return type thing

File: src/pythie_serving/table_wrapper.py (hunk 3 above)
Comment (Yax94) [reply src/pythie_serving/table_wrapper.py:23]: I think this is also a mypy configuration issue. I have no warning when running mypy in local

**Review by matbleu — APPROVED** [review]

**Comment by matbleu** [comment]
@Yax94 j'ai validé rapidement (pour faire un test de CI) mais tu peux juste rebase avant que je merge stp ? :) et c'est tout bon

**Pull request merged**